src/gui/date/DatePicker.ts

```diff
@@ -414,20 +422,23 @@ export class VisualDatePicker implements Component<VisualDatePickerAttrs> {
 	}
 
 	private renderDay({ date, day, isPaddingDay }: CalendarDay, attrs: VisualDatePickerAttrs): Children {
-		const size = px(this.getElementWidth(attrs))
 		const eventForDay = attrs.eventsForDays?.get(date.getTime())
 		const hasEvent = eventForDay && eventForDay.length > 0
 		const isExpandableDatePicker = attrs.isExpandableDatePicker ?? false
+		const size = this.getElementWidth(attrs)
 		const isSelectedDay = isSameDayOfDate(date, attrs.selectedDate)
+		const dateIndicator =
+			isPaddingDay && !isExpandableDatePicker
+				? ""
+				: getDateIndicator(date, attrs.selectedDate).replace("accent-bg", isExpandableDatePicker ? "accent-bg-translucent" : ".accent-bg")
```

Comment:
why would you replace the selector?

not only string manipulation on selectors is extremely dangerous (it's a bit safer using `class` prop) but also in this case really pointless, just inline this function